gradle-incremental-configuration-cache/src/main/java/com/palantir/gradle/configcache/incremental/IncrementalConfigurationCachePlugin.java

```diff
@@ -37,19 +55,23 @@ public final void apply(Project project) {
 
         Path allowListPath = project.getRootProject().getProjectDir().toPath().resolve(ALLOW_LIST_FILE);
         if (!Files.exists(allowListPath)) {
-            throw new RuntimeException(
-                    "Configuration cache allowed tasks file not found at %s".formatted(allowListPath));
+            throw new RuntimeException("Configuration cache allow list file not found at %s\n%s"
+                    .formatted(allowListPath, ALLOW_LIST_INFO));
         }
 
         AllowListFile allowList = new AllowListFile(allowListPath);
         Set<String> enabledTasks = allowList.loadAllowedTasks();
-
         project.getAllprojects().forEach(proj -> proj.getTasks().configureEach(task -> {
             if (!enabledTasks.contains(task.getPath())) {
                 task.notCompatibleWithConfigurationCache(
-                        "Configuration cache is not enabled for this task, as it was not included in %s"
-                                .formatted(ALLOW_LIST_FILE));
+                        "Configuration cache is not enabled for this task, as it was not included in %s\n%s"
+                                .formatted(allowListPath, ALLOW_LIST_INFO));
```

Comment:
I included this change here because, while drafting the PR, I was thinking about user's experience with our error and reassurance messages. I kept it in this PR to help manage our current review workload.

Please let me know if you’d prefer this as a separate PR.